backend/README.md

```diff
@@ -7,141 +7,112 @@ This directory contains the backend implementation for the Summa Proof of Solven
 ```
-1. Ownership proofs are submitted successfully!
-```
-
-### 2. Submit Commitment
-
-The CEX must submit a commitment to the Summa contract for each round. This commitment consists of a timestamp, a SNARK proof, a grand sum proof and total balances.
-
-Without CEX publishing the commitment, users cannot verify their inclusion proof on the Summa contract. This is because the inclusion verifier function internally requires the SNARK proof, but users only know the timestamp associated with the round and the verifier functions does not requre the SNARK proof directly.
 
-In here, we'll introduce you through the process of submitting a commitment using the `Round` to the Summa contract.
-The Round serves as the core of the backend in Summa, and we have briefly described it in the Components section.
+### 2. Generating and Exporting Inclusion Proofs
 
-To initialize the `Round` instance, you'll need paths to the liabilities CSV file (`entry_16.csv`) and the `ptau/hermez-raw-17` file. The files serve the following purpose:
+Assuming you're a CEX, after committing the commitment publicly, you should generate inclusion proofs for every user. This proof verifies the presence of specific elements in the polynomials encoding the username and balances.
 
-- `entry_16.csv`: contains the username and liabilities entries for each CEX user (necessary to build the commitment). Liabilities column names have the following format: `balance_<CRYPTOCURRENCY>_<CHAIN>`, where <CRYPTOCURRENCY> and <CHAIN> are the names of the cryptocurrencies and their corresponding blockchains. <CHAIN> values are the same as in the Address Ownership Proof step;
-- `ptau/hermez-raw-17`: contains parameters for constructing the zk circuits.
-
-Using the `Round` instance, the solvency proof is dispatched to the Summa contract with the `dispatch_solvency_proof` method.
-
-If this step successfully ran, you can see this message:
-
-```
-2. Solvency proof is submitted successfully!
-```
-
-### 3. Generating and Exporting Inclusion Proofs
-
-Assuming you're a CEX, after committing the commitment and ownership proofs to the Summa contract, you should generate inclusion proofs for every user. This proof verifies the presence of specific elements in the polynomials encoding the username, balances.
-
-After generating the inclusion proof, it's transformed into a JSON format for easy sharing.
+After generating the inclusion proof, it is transformed into a JSON format for easy sharing.
 
 Upon successful execution, you'll find a file named `user_0_proof.json` and see the following message:
 
+```bash
+2. Exported proof to user #0, as `user_0_proof.json`, with verifier params `verifier_params.json`
 ```
-3. Exported proof to user #0, as `user_0_proof.json`
-```
 
-### 4. Verify Proof of Inclusion
+Note that the `verifier_params.json` file can be used in any other round unless the same circuit configurations, such as `N_CURRENCIES` and `K`.
```

Comment:
Do you mean that it can be re-used if N_CURRENCIES and K didn't change?

Reply:
Yes, exactly!